ui/src/app/components/webcomponents/test-case-result-filter.component.ts

```diff
@@ -26,21 +28,41 @@ export class TestCaseResultFilterComponent implements OnInit {
   public filterTestCasePriorities: string[];
   public testcasePrioritiesList: Page<TestCasePriority>;
   public testCaseTypesList: Page<TestCaseType>;
+  public filterTagIds: number[];
+  public tags: TestCaseTag[];
+  public createdDateRange = new FormGroup({
+    start: new FormControl(),
+    end: new FormControl()
+  });
+  public updatedDateRange = new FormGroup({
+    start: new FormControl(),
+    end: new FormControl()
+  });
+  maxDate = new Date();
 
   constructor(
     public translate: TranslateService,
     private testCaseTypeService: TestCaseTypesService,
     private testCasePriorityService: TestCasePrioritiesService) {
   }
 
+  dateInvalid(DateRange) {
```

Comment:
it can be named as isDataInvalid to denote conditional check

Reply:
okay bro, will change

Reply:
can we make this isInvalidDateRange which express better actually